resources/axon.rb

```diff
@@ -58,9 +58,30 @@
 
   # Set platform specific variables
   case node['platform']
-  when 'centos', 'redhat', 'suse', 'oraclelinux', 'oracle', 'amazon'
+  when 'centos', 'redhat', 'suse', 'oraclelinux', 'amazon'
     ext = '.rpm'
     eg_service_name = 'tw-eg-service'
+  when 'oracle'
+    ext = '.rpm'
+    eg_service_name = 'tw-eg-service'
+    # for Oracle UEK, need to run few extra commands as pre-requisites
+    if node["hostnamectl"]["kernel"].include?('uek')
+      if node['platform_version'] .include?('7.')
+        pre_requisite_cmds = "yum -y install kernel-uek-devel-`uname -r` && sudo yum install yum-utils -y && "\
+        "sudo curl -O https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && "\
+        "sudo yum clean metadata dbcache && sudo yum install epel-release-latest-7.noarch.rpm -y && "\
+        "sudo yum-config-manager --disable epel && sudo yum --enablerepo=epel install dkms -y"
+      else
+        pre_requisite_cmds = "yum -y install kernel-uek-devel-`uname -r` && sudo yum install yum-utils -y && "\
+        "sudo curl -O https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm && "\
+        "sudo yum clean metadata dbcache && sudo yum install epel-release-latest-8.noarch.rpm -y && "\
+        "sudo yum-config-manager --disable epel && sudo yum --enablerepo=epel install dkms -y"
+      end
+
+      execute 'test' do
```

Comment:
This could be better named. Like "Install dkms prerequisites".

Reply:
Done